scripts/visualise_results.py

```diff
@@ -0,0 +1,80 @@
+import os
+from glob import glob
+from datetime import datetime
+
+import statistics
+import json
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+def get_stats(folders):
+    results_list = []
+
+    for folder in folders:
+
+        get_request_response_times = []
+        post_request_response_times = []
+        all_response_times = []
+
+        for file in os.listdir(folder):
+            if 'distribution.csv' not in file:
+                continue
+
+            with open(f'{os.getcwd()}/{folder}/{file}') as f:
+                data = f.read()
```

Comment:
`glob` can be used to match filenames. This also has the advantage it returns the full file path, so it makes the `open` command clearer too.

```python
for file in glob(folder + '/*distribution.csv'):

    with open(file) as f:
        data = f.read()
```